build.gradle

```diff
@@ -4,15 +4,26 @@ buildscript {
     repositories {
         jcenter()
         mavenCentral()
+        maven { url 'https://maven.fabric.io/public' }
```

Comment:
@lorenzoPrimi We should remove the entry for Fabric. Otherwise f-droid will not build.